server/request_record.go

```diff
@@ -0,0 +1,48 @@
+package server
+
+import (
+	"context"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/flashbots/rpc-endpoint/types"
+	"github.com/google/uuid"
+	"net/http"
+	"time"
+)
+
+type RequestRecord struct {
+	logger            log.Logger
+	requestEntry      *types.RequestEntry
+	ethSendRawTxEntry *types.EthSendRawTxEntry
+}
+
+func NewRequestRecord() *RequestRecord {
+	// If the incoming request exited in the server then below id will be used for RequestEntry
+	// If not, below id will be replaced by uid if the request forwarded to request handler
+	id := uuid.New()
```

Comment:
id will be set by the database, not needed here

Reply:
Yes.added this just for logging. will be updated while adding db integration

Reply:
Like to keep the id field same as uid so that it will make the log comparision easier during debugging.